src/models/schemas.py

```
@@ -0,0 +1,612 @@
+        result['income_household_total_yearly'] = result['income_household_total_monthly'] * 12.0
+        
+        # Income less foster children
+        income_less_foster = 0.0
+        for i, person in enumerate(persons):
+            if person.household_member_type != HouseholdMemberType.FOSTER_CHILD:
+                income_less_foster += result['income_person_monthly'].get(i, 0.0)
+        result['income_household_total_monthly_less_foster'] = income_less_foster
+        
+        # Income less gifts
+        income_less_gifts = 0.0
+        for i, person in enumerate(persons):
```

Ruff (0.12.2)

Loop control variable `person` not used within loop body

(B007)